spec/unit/iq_client_spec.rb

```diff
@@ -44,7 +44,8 @@
         public_application_id: 'appid',
         server_url: 'server_url',
         username: 'iquser',
-        auth_token: 'iqpass'
+        auth_token: 'iqpass',
+        stage: 'iqstage'
```

Comment:
Might be good to use a real value here, since it's effectively an enum on the IQ Server side?

Reply:
I was on the fence about using a 'real' stage value, but I went with the "fake" value to make it clear we're not validating the actual value in the tests.

Reply:
That's fine, my preference is always to use real values in tests (if the other side uses them), but I often break that rule, so no worries :)